### PR TITLE
{bio}[foss/2018b] CD-HIT v4.6.8

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/51552111067f027e4f558e13992b43cf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e2fe669ef5ec1fcbf52ad9081c09110b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c4904e1c5985e77a68e2324111670522 for a full test report.

File: easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.8-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# make sure compilation flags are passed down (e.g. to enable OpenMP)
+buildopts = ' CC="$CXX" CCFLAGS="$CPPFLAGS $CXXFLAGS"'
+
+# put here the list of generated executables when compiling
+list_of_executables = ['cd-hit', 'cd-hit-est', 'cd-hit-2d', 'cd-hit-est-2d', 'cd-hit-div', 'cd-hit-454']
+
+# this is the real EasyBuild line to copy all the executables and perl scripts to "bin"
+files_to_copy = [(list_of_executables, 'bin'), (['*.pl'], 'bin'), 'README', 'doc', 'license.txt']
+
+postinstallcmds = ["sed -i 's@#!/usr/bin/perl@/usr/bin/env perl@' %(installdir)s/bin/*.pl"]
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.8-foss-2018b.eb]: there is no dependency on perl?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3bad78b0cf72c5e62db3e503191a9878 for a full test report.

**Pull request merged**